docker-compose.yml

```
@@ -1,58 +1,53 @@
 services:
-  slimevr-database:
-    image: 'postgres'
+  db:
+    image: "postgres"
     restart: always
     environment:
       POSTGRES_USER: ${POSTGRES_USER}
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-      POSTGRES_DB: ${POSTGRES_DATABASE}
+      POSTGRES_DB: ${POSTGRES_DB}
+    expose:
+      - 5432
     volumes:
       - database-volume:/var/lib/postgresql/data
     networks:
       - slimevr-network
-  slimevr-api:
+  api:
     build:
       context: .
```

Add secretes to build env
```suggestion
      secrets:
        - access_key
        - secret_key
      context: .
```